test/test_hydrostatic_free_surface_models.jl

```diff
@@ -97,12 +97,37 @@ topos_3d = ((Periodic, Periodic, Bounded),
             for arch in archs, FT in float_types
                 grid = RectilinearGrid(arch, FT, topology=topo, size=(1, 1, 1), extent=(1, 2, 3))
                 model = HydrostaticFreeSurfaceModel(grid=grid)
-
                 @test model isa HydrostaticFreeSurfaceModel
             end
         end
     end
 
+    @testset "Halo size check in model constructor" begin
+        grid = RectilinearGrid(arch, FT, topology=topo, size=(1, 1, 1), extent=(1, 2, 3))
```

Comment:
@glwagner, which `topo` here?

did you mean to have this as part of a loop?

Reply:
No, just forgot to change that... anything is fine, so maybe `topo=(Periodic, Periodic, Bounded)`.

Reply:
Haha.
So I did 32e3e40 which does the testing for all topos! Too much?

Reply:
OK. I see what you did with `topos` -> `topos_3d`.